docs/unified-test-documentation/dasharo-performance/402-cpu-frequency.md

```diff
@@ -143,6 +143,7 @@ expected frequency.
     cat /proc/cpuinfo | grep -i mhz
     ```
 
+1. Repeat command couple times
```

Comment:
```suggestion
1. Repeat command a couple of times
```

Reply:
I'm in favor of procedures being strict. We can intruct to run it e.g. 5 times with 1s interval. Or (maybe even better) provide a one-liner loop which does just his. The less space for interpretation in procedures, the better.